libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Selectors/Microsoft.MostSpecificSelector.schema

```diff
@@ -0,0 +1,12 @@
+﻿{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/botbuilder-tools/SchemaGen/packages/DialogSchema/src/dialogSchema.schema",
+    "$role": "unionType(Microsoft.IRuleSelector)",
+    "title": "Select most specific true rule",
```

Comment:
>Select most specific true rule [](start = 14, length = 30)

Best Match ?

Reply:
Select Best

---
In reply to: [274720932](https://github.com/Microsoft/botbuilder-dotnet/pull/1655#discussion_r274720932) [](ancestors = 274720932)